docker/build-tools/Dockerfile

```diff
@@ -119,9 +121,9 @@ RUN GO111MODULE=on go get k8s.io/code-generator/cmd/go-to-protobuf@kubernetes-${
 ADD https://github.com/upx/upx/releases/download/v${UPX_VERSION}/upx-${UPX_VERSION}-amd64_linux.tar.xz /tmp
 RUN tar -xJf upx-${UPX_VERSION}-amd64_linux.tar.xz -C /tmp
 RUN mv /tmp/upx-${UPX_VERSION}-amd64_linux/upx /usr/bin
-RUN upx --lzma /go/bin/*
-RUN mkdir -p ${OUTDIR}/go/bin
-RUN mv /go/bin/* ${OUTDIR}/go/bin
+RUN upx --lzma /gotools/bin/*
+#RUN mkdir -p ${OUTDIR}/go/bin
```

Comment:
Do we still need to two commented-out lines?

Reply:
no.